opendatahub/odh-manifests/model-mesh_stable/odh-model-controller/overlays/odh/odh_model_controller_manager_patch.yaml

```diff
@@ -11,11 +11,6 @@ spec:
             - --leader-elect
             - "--monitoring-namespace"
             - "$(MONITORING_NS)"
-#This is hardcoded to be false in the overlay to model-mesh. 
-# Once Kserve and Modelmesh CRD conflict is resolved and both components can be installed together, 
-# ODH Model Controller will not be deployed as a overlay anymore and will be deployed only as an independent component. 
-# When that happens, the "kserve-enabled" flag needs to be removed
-            - --kserve-enabled=false            
```

Comment:
Why is this removed? This is needed for odh-model-controller when used with modelmesh.

Reply:
Now modelmesh-serving and kserve could be deployed togeather. I had tested it on my system and its working.

Reply:
added back